apps/src/templates/levelSummary/FreeResponseResponses.jsx

```diff
@@ -44,26 +66,26 @@ const FreeResponseResponses = ({responses, showStudentNames, eventData}) => {
           )}
         >
           <p>{response.text}</p>
-          <ResponseMenuDropdown
-            response={response}
-            hideResponse={userId => {
-              analyticsReporter.sendEvent(
-                EVENTS.CFU_RESPONSE_HIDDEN,
-                eventData,
-                PLATFORMS.BOTH
-              );
-              setHiddenResponses(prevHidden => [...prevHidden, userId]);
+          <ActionDropdown
+            name="free-response"
+            menuPlacement="right"
+            labelText="Free Response"
```

Comment:
how is this used

Reply:
labelText would get set as the aria-label, updated this to be at parity with previous implementation